pkg/infrastructure/infinispan.go

```diff
@@ -49,3 +53,49 @@ func IsInfinispanOperatorAvailable(cli *client.Client, namespace string) (bool,
 	log.Debugf("Looks like Infinispan Operator is not available in the namespace %s", namespace)
 	return false, nil
 }
+
+// GetInfinispanServiceURI fetches for the Infinispan service linked with the given KogitoInfra and returns a formatted URI
+func GetInfinispanServiceURI(cli *client.Client, infra *v1alpha1.KogitoInfra) (uri string, err error) {
+	if &infra == nil || &infra.Status == nil || &infra.Status.Infinispan == nil {
+		return "", nil
+	}
+
+	service := &corev1.Service{
+		ObjectMeta: metav1.ObjectMeta{Name: infra.Status.Infinispan.Service, Namespace: infra.Namespace},
+	}
+	exists := false
+	if exists, err = kubernetes.ResourceC(cli).Fetch(service); err != nil {
+		return "", err
+	}
+
+	if exists && len(service.Spec.Ports) > 0 {
+		return fmt.Sprintf("%s:%d", service.Name, service.Spec.Ports[0].TargetPort.IntVal), nil
+	}
+
+	return "", nil
+}
+
+// GetInfinispanCredentials fetches the Infinispan secret credentials managed by the given KogitoInfra and returns the credentials stored
+func GetInfinispanCredentials(cli *client.Client, infra *v1alpha1.KogitoInfra) (user, password string, err error) {
+	user = ""
+	password = ""
+	err = nil
+	if &infra == nil || &infra.Status == nil || &infra.Status.Infinispan == nil {
+		return
+	}
+
+	secret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{Name: infra.Status.Infinispan.CredentialSecret, Namespace: infra.Namespace},
```

Comment:
here no credentials are set ... See screenshot
![image](https://user-images.githubusercontent.com/17157711/69720380-c1feb480-1112-11ea-8077-dc93a84b040e.png)

Reply:
Which Infinispan operator are you using? I guess we tested with different versions or the operator has been updated. Please paste the output for “oc get secrets”

Reply:
```
$ oc get secrets
NAME                                      TYPE                                  DATA   AGE
builder-dockercfg-cl7br                   kubernetes.io/dockercfg               1      157m
builder-token-9jlr2                       kubernetes.io/service-account-token   4      157m
builder-token-dxlbf                       kubernetes.io/service-account-token   4      157m
default-dockercfg-cmlrc                   kubernetes.io/dockercfg               1      157m
default-token-npwz2                       kubernetes.io/service-account-token   4      157m
default-token-vmzwl                       kubernetes.io/service-account-token   4      157m
deployer-dockercfg-j6f8v                  kubernetes.io/dockercfg               1      157m
deployer-token-vsn6k                      kubernetes.io/service-account-token   4      157m
deployer-token-zp4qb                      kubernetes.io/service-account-token   4      157m
infinispan-operator-dockercfg-dwl8p       kubernetes.io/dockercfg               1      95m
infinispan-operator-token-hmfdl           kubernetes.io/service-account-token   4      95m
infinispan-operator-token-zcgtl           kubernetes.io/service-account-token   4      95m
kogito-cloud-operator-dockercfg-67n5w     kubernetes.io/dockercfg               1      95m
kogito-cloud-operator-token-mdmvv         kubernetes.io/service-account-token   4      95m
kogito-cloud-operator-token-sqdm8         kubernetes.io/service-account-token   4      95m
kogito-infinispan-app-generated-secret    Opaque                                2      93m
kogito-infinispan-credential              Opaque                                2      93m
kogito-infinispan-mgmt-generated-secret   Opaque                                2      93m
kogito-service-viewer-dockercfg-cq6f4     kubernetes.io/dockercfg               1      95m
kogito-service-viewer-token-4jj95         kubernetes.io/service-account-token   4      95m
kogito-service-viewer-token-9jslc         kubernetes.io/service-account-token   4      95m
```

Reply:
Yup, they changed the secret. Could you give me the yaml output for:

```
kogito-infinispan-app-generated-secret    Opaque                                2      93m
kogito-infinispan-credential              Opaque                                2      93m
kogito-infinispan-mgmt-generated-secret   Opaque                                2      93m
```

Reply:
In this version they had two secrets, I'll fetch for `kogito-infinispan-app-generated-secret` instead.. just describe it to me please

Reply:
@radtriste https://github.com/infinispan/infinispan-operator/blob/2870aaa665308dce507944ce2707d205d58e7176/pkg/controller/infinispan/infinispan_controller.go#L464